docs/source/conf.py

```diff
@@ -0,0 +1,134 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+from recommonmark.transform import AutoStructify
+
+sys.path.insert(0, os.path.abspath(".."))
+import qibo_tii_provider
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "qibo-tii-provider"
+copyright = "The Qibo team"
+author = "The Qibo team"
+release = qibo_tii_provider.__version__
+
+# -- General configuration ---------------------------------------------------
+#
+# https://stackoverflow.com/questions/56336234/build-fail-sphinx-error-contents-rst-not-found
+master_doc = "index"
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.doctest",
+    "sphinx.ext.coverage",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
+    "sphinx_copybutton",
+    "recommonmark",
+    "nbsphinx",
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# Markdown configuration
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = {".rst": "restructuredtext", ".txt": "markdown", ".md": "markdown"}
+
+autosectionlabel_prefix_document = True
+# Allow to embed rst syntax in  markdown files.
+enable_eval_rst = True
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+
+html_theme = "furo"
+html_favicon = "favicon.ico"
+
+# custom title
+html_title = "Qibo TII Provider · v" + release
```

Comment:
Could you please just copy the layout from Qibo? The title and navigation bar should match the one from Qibo too.

Reply:
Do you want to show `Qibo` in the sidebar even if the documentation is for the `Qibo client` package?

Reply:
Yes, it will become the standard approach for external providers.

Reply:
ok, fine